flatpak/org.nickvision.tubeconverter.json

```diff
@@ -172,7 +172,7 @@
                 }
             ]
         },
-        "libsecret/libsecret.json",
+        "libsecret.json",
```

Comment:
What I meant it why have a separate `libsecret.json` file all together? Just paste it inside the manifest itself...

Reply:
Ah, okay